flink-python/src/main/java/org/apache/flink/table/runtime/typeutils/serializers/StringSerializer.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.runtime.typeutils.serializers;
+
+import org.apache.flink.annotation.Internal;
+import org.apache.flink.api.common.typeutils.SimpleTypeSerializerSnapshot;
+import org.apache.flink.api.common.typeutils.TypeSerializerSnapshot;
+import org.apache.flink.api.common.typeutils.base.TypeSerializerSingleton;
+import org.apache.flink.core.memory.DataInputView;
+import org.apache.flink.core.memory.DataOutputView;
+import org.apache.flink.table.runtime.util.StringUtf8Utils;
+
+import java.io.IOException;
+
+/**
+ * <p>We create the StringSerializer instead of using the StringSerializer of flink-core module
+ * because the StringSerializer of flink-core module serialize every Char of String in serialize
+ * method and deserialize the Char to build the String. We want to convert String to UTF-8 bytes
+ * to serialize which is compatible with BinaryStringSerializer in blink.</p>
+ *
+ * <p>So we create this StringSerializer (only used in Java and Python data communication in udf).
+ *
+ * <p>StringSerializer for String.
+ */
+@Internal
+public class StringSerializer extends TypeSerializerSingleton<String> {
+
+	private static final long serialVersionUID = 1L;
+
+	/**
+	 * Sharable instance of the StringSerializer.
```

Comment:
Remove this comment as it seems unnecessary. Besides, this is to be consistent with the DateSerializer which also has no such comment.